src/modules/emails/emails.service.test.ts

```diff
@@ -0,0 +1,42 @@
+import { PostEmailsGenerator } from '@ukef-test/support/generator/post-emails-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { resetAllWhenMocks, when } from 'jest-when';
+
+import { GovukNotifyService } from '../../helper-modules/govuk-notify/govuk-notify.service';
+import { EmailsService } from './emails.service';
+
+jest.mock('@ukef/modules/informatica/informatica.service');
+
+describe('EmailsService', () => {
+  const valueGenerator = new RandomValueGenerator();
+
+  let service: EmailsService;
+  let govukNotifyServiceSendEmail: jest.Mock;
+
+  beforeEach(() => {
+    govukNotifyServiceSendEmail = jest.fn();
+    const govukNotifyService = new GovukNotifyService();
+    govukNotifyService.sendEmail = govukNotifyServiceSendEmail;
+    resetAllWhenMocks();
+
+    service = new EmailsService(govukNotifyService);
+  });
+
+  describe('sendEmail', () => {
+    const govUkNotifyKey = valueGenerator.string();
+    const {
+      requests: [request],
+      postEmailsResponse,
+    } = new PostEmailsGenerator(valueGenerator).generate({
+      numberToGenerate: 1,
+    });
+
+    it('returns email sent receipt from the backend service', async () => {
```

Comment:
Can we add an asswertion where `service.sendEmail(null, null)` please.

Reply:
EmailsService is just a wrapper for GovUK Notify Service, we have similar pattern in other MDM endpoints.

We don't test missing data in services, as data validation happends at begining of request or is done by typescript.